nav2_system_tests/src/localization/README.md

```diff
@@ -0,0 +1,14 @@
+# Localization Testing
+
+The intention of the localization test is to ensure robot’s pose and transforms are available.
+
+Currently, only a simple test that checks the `initialpose` has been implemented.  The `test_localization` module publishes an initial pose on `initialpose` topic and then it listens to `amcl_pose` topic. If the `amcl_pose` is similar to `initial pose` within a predefined tolerance the test passes.
+
+To run the test, first the GAZEBO_MODEL_PATH variable needs to be set and then ctest commend can be used to initiate the test.  
+
+1. Open a terminal and set `export GAZEBO_MODEL_PATH= /turtlebot3_ws/src/turtlebot3_simulations/turtlebot3_gazebo/models`
```

Comment:
If I'm not mistaken, this variable needs to be set before building, not just before testing.

Reply:
Yes, Carl is correct, it needs to be set before 'colcon build --symlink-install' is run. You should add that as step 2 and then run the test as step 3.